harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/localfilesystem/LocalFilesystemHarvester.java

```diff
@@ -284,4 +287,19 @@ protected void doUpdate(String id, Element node) throws BadInputEx, SQLException
 
     }
 
+    private void runBeforeScript() throws IOException, InterruptedException {
+		if (params.beforeScript == null) {
```

Comment:
StringUtils.isEmpty() instead ?

Because of the following code below (diff in LocalFileSystemParams.java):

```diff
+        beforeScript = Util.getParam(site, "beforeScript", "");
```